.github/workflows/ci.yml

```diff
@@ -128,3 +128,23 @@ jobs:
           cmake .. -G "MinGW Makefiles" -DBUILD_TESTING=ON -DBUILD_TESTING_INTEGRATION=ON -DC_WARNINGS_AS_ERRORS=ON -DCPP_WARNINGS_AS_ERRORS=ON
           cmake --build .
           ctest --verbose -R "allocator|value|example_basic|integration_basic"
+
+  build_and_test_linux_wasm:
+    strategy:
+     matrix:
+        platform: [ubuntu-18.04]
+    runs-on: ${{ matrix.platform }}
+    env:
+      MG_VERSION: "1.4.0"
+    steps:
+      - name: Set-up repository
+        uses: actions/checkout@v2
+      - name: Install environment
+        run: |
+          sudo apt install -y clang-8 clang++-8
+
+      - name: Build with clang
+        run: |
+          mkdir build && ./wasm/install_deps.sh && ./wasm/activate.sh && cd build
```

Comment:
Does it make sense to move `install_deps.sh` in the action above, since it also installs environment?

Reply:
I removed the step above as it's no longer needed.